app/models.py

```diff
@@ -890,7 +894,7 @@ def __init__(self,
                                              date_modified)
         self.dtype = dtype
 
-        if dtype != determination_type.ACKNOWLEDGMENT:
+        if dtype != determination_type.ACKNOWLEDGMENT and dtype != determination_type.REOPENED:
```

Comment:
to shorten this you can check if the the determination type is present in a list